standard-covoiturage_openapi.yaml

```diff
@@ -641,9 +819,10 @@ components:
         - type: object
           required:
             - passenger
+            - webUrl
           properties:
-            passenger:
-              $ref: '#/components/schemas/User'
+            passengerId:
```

Comment:
This change is not compatible with the booking events flow (https://github.com/fabmob/standard-covoiturage/pull/5) as the passenger / driver will not be known on MaaS's side.